### PR TITLE
feat: added solution to who likes it (Go)

### PR DESCRIPTION
**Pull request by aldotele**

### CODE REVIEW

File: challenges/easy/who-likes-it/solutions/aldotele/main.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    var person string
+    n, err := fmt.Scanf("%s", &person)
+    if n == 1 {
+        x = append(x, person)
+    }
+    return enterLikes(x, err)
+}
+
+func makeSentence(likes []string) string {
+  if len(likes) == 1 {
```
Comment (micheleriva) [inline challenges/easy/who-likes-it/solutions/aldotele/main.go:24]: Avoid computing slice length on each branch. You can declare a variable such as `likesCount := len(likes)` and use it for each condition. Also, using `switch/case` here could lead to cleaner code

File: challenges/easy/who-likes-it/solutions/aldotele/main.go [hunk 2]
```diff
@@ -0,0 +1,35 @@
+package main
+
+import "fmt"
+
+func main() {
+  fmt.Println("Enter people below (blank to quit)")
```
Comment (micheleriva) [inline challenges/easy/who-likes-it/solutions/aldotele/main.go:6]: can you please write an example of how to use this program? Thanks!

Comment (aldotele) [reply challenges/easy/who-likes-it/solutions/aldotele/main.go]: I changed approach to make it easier and self-explanatory, hope it is more clear now!

File: challenges/easy/who-likes-it/solutions/aldotele/main.go (hunk 1 above)
Comment (aldotele) [reply challenges/easy/who-likes-it/solutions/aldotele/main.go]: Thanks for the advice, I use switch/case now!

**Pull request merged**